subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -754,10 +752,7 @@ static int wifi_scan_args_to_params(const struct shell *sh,
 static int cmd_wifi_scan(const struct shell *sh, size_t argc, char *argv[])
 {
 	struct net_if *iface = net_if_get_first_wifi();
-	struct wifi_scan_params params = { .dwell_time_active =
-					    WIFI_MGMT_SCAN_DEFAULT_DWELL_TIME_ACTIVE,
-					   .dwell_time_passive =
-					    WIFI_MGMT_SCAN_DEFAULT_DWELL_TIME_PASSIVE, };
+	struct wifi_scan_params params = { 0 };
```

Comment:
What happens now if the driver if the dwell active/passive times are set to default value of 0 i.e., does this mean that user always needs to set some value for the dwell time?

Reply:
It's advance scan option, so if user wants, it can be set. Otherwise we are removing the constrains, just checking for positive value.

Reply:
My point was that I suppose then the 0 value is considered invalid and ignored by the driver then?

Reply:
The 0 value will be taken as driver can choose it's own default values, as it's invalid.